buildSrc/src/main/java/com/osacky/flank/gradle/FladlePluginDelegate.kt

```diff
@@ -119,7 +119,7 @@ class FladlePluginDelegate {
               project.log("Configuring fladle.debugApk from variant ${this@app.name}")
               extension.debugApk.set(this@app.outputFile.absolutePath)
             }
-            if (extension.roboScript == null && !extension.instrumentationApk.isPresent) {
+            if (!extension.roboScript.isPresent && !extension.instrumentationApk.isPresent) {
```

Comment:
in order to maintain the same logic as before, i think this should be:
```suggestion
            if (extension.roboScript.orNull == null && !extension.instrumentationApk.isPresent) {
```